cv32e20/sim/ExternalRepos.mk

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+# Variables to generate the command to clone external repositories.
+# For each repo there are a set of variables:
+#      *_REPO:   URL to the repository (note, not all are in GitHub).
+#      *_BRANCH: Name of the branch you wish to clone;
+#                Set to 'master' to pull the master branch.
+#      *_HASH:   Value of the specific hash you wish to clone;
+#                Set to 'head' to pull the head of the branch you want.
+#
+
+export SHELL = /bin/bash
+
+CV_CORE_REPO   ?= https://github.com/MikeOpenHWGroup/cve2
```

Comment:
assuming that this needs to be updated to the dev branch ?

Reply:
Good eye, but no. :wink:

Currently, there is no `dev` branch on [openhwgroup/cve2](https://github.com/openhwgroup/cve2/) and my [PR for that repo](https://github.com/openhwgroup/cve2/pull/16) goes to the `main` branch.  Once that PR is merged in, I will issue another PR for core-v-verif to set `CV_CORE_REPO` to https://github.com/openhwgroup/cve2

Reply:
ok, accepted